docs/docs/aleph_zero/sdk.md

```diff
@@ -10,31 +10,55 @@ SDK (Software Development Kit) can be used to interact with our DEX programatica
 
 ### Download
 
-Published package can be found here: [NPM](https://google.com/).
+Published package can be found [here](https://google.com/).
 
 ### Build
 
-You can find build steps here: [Installation](installation.md).
+You can find build steps [here](installation.md).
 
 ## Overview
 
-### Structure
+The Invariant SDK comprises three distinct contracts:
 
-Invariant SDK consists of 3 separate contracts: our DEX contract (Invariant), token contract (PSP22), contract that can be used to wrap native currency (Wrapped AZERO), various values and helper functions. You can deploy your own or use an existing contract.
+1. DEX Contract (Invariant): This is the core contract handling DEX functionality within the Invariant ecosystem.
 
-### Txs and Queries
+2. Token Contract (PSP22): This contract is responsible for managing and implementing the PSP22 token standard within the Invariant framework. Allows to deploy or load existing contracts.
 
-After you load or deploy a contract you can start interacting with it. In order to make a contract call you have to use methods of class that represents specific contract. First parameter will always be an account you want to use and rest of parameters will be an entrypoint parameters.
+3. Wrapped Native Currency Contract (Wrapped AZERO): This contract allows users to wrap the native currency, providing compatibility and interoperability within the Invariant ecosystem.
 
-### Event listeners
+### Transactions and Queries
 
-Invariant class has additional methods that allows you to attach event listeners so you can run your own code based on contract activity.
+When working with contracts, developers can seamlessly initiate interactions by calling methods from the corresponding contract class. The first parameter designates the account, and subsequent parameters act as entrypoint parameters, facilitating smooth contract interactions.
+
+1. **Transactions**: These involve invoking methods that result in changes to the blockchain state. Transactions typically alter the data stored on the blockchain and may include operations like transferring assets, updating records, or executing specific actions.
```

Comment:
Beyond, considering tx and query distinctions, within the realm of blockchain terminology, pay closer attention to the differences from SDK perspective - that is, the interface distinctions between these two call types. For instance, one notable distinction is that each tx return the **hash** of the confirmed transaction, this field absent in queries. Please describe more differences as you come across them (interfaces perspective)